src/components/setup/addon.tsx

```diff
@@ -1,11 +1,9 @@
-import { Tag } from "@components/global/tag.module";
+import styles from "./setup_item.module.scss";
+import { Tag } from "@components/global/tag";
 import { faker } from "@faker-js/faker";
-import prisma from "@prisma/client";
+import { RouterOutput } from "@server/routers/_app";
```

Comment:
This type should be exported from  `/utils/trpc`, importing from server files in components kinda nono

Reply:
There's also RouterInputs we should type
```ts
/**
 * Inference helper for inputs
 * @example type HelloInput = RouterInputs['example']['hello']
 **/
export type RouterInputs = inferRouterInputs<AppRouter>;
/**
 * Inference helper for outputs
 * @example type HelloOutput = RouterOutputs['example']['hello']
 **/
export type RouterOutputs = inferRouterOutputs<AppRouter>;
```